### PR TITLE
Update release notes for v1.6.1

### PR DESCRIPTION
**Pull request by MikeHart85**

### CODE REVIEW

File: docs/source/release_notes.rst
Comment (mrakitin) [inline docs/source/release_notes.rst:13]:
```suggestion
  via ``set_defaults``, instead of always defaulting to 1.0 second.
```

**Review by mrakitin — APPROVED** [review]
Looks good. Minor suggestion.

File: docs/source/release_notes.rst
Comment (MikeHart85) [reply docs/source/release_notes.rst:13]: It's a bit counter-intuitive, but I think the way I have it is grammatically correct. 

Only an integer 1 can be singular, while all floating point numbers are plural. So it would have to either be "1 second" or "1.0 seconds"... though I have no strong preference to which of the two to use. I picked 1.0 because that is exactly as it is in the code.

Comment (mrakitin) [reply docs/source/release_notes.rst:13]: That's a good point. I think the release notes are for humans, not computers, maybe having "1 second" makes sense here. I'll update my suggestion.

**Review by mrakitin — APPROVED** [review]

**Comment by mrakitin** [comment]
@MikeHart85, do you want to incorporate #963 into the release notes too?

**Comment by MikeHart85** [comment]
Yes, adding that as well. 

I think we should also merge in #969 if possible. The change looks good to me, given the discussion in the issue, but another pair of eyes would be great. Could someone else also have a look and hit the merge button?

**Comment by danielballan** [comment]
Good point. Done.

**Comment by MikeHart85** [comment]
I think this is ready to merge, unless there are any other PRs we want to include?

If everything looks ready to go, I'll proceed with the release as soon as someone merges these notes.

**Review by danielballan — APPROVED** [review]

**Pull request merged**
